### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Create Web Site-Add Bindings_Error Identification: Error identification is not proper when invalid value is provided in the form fields.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#148]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS3.3.1

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "Create Web Site" expandable button and Press Enter.
6. Navigate to "bindings" page tab.
7. Tab till "Add" button and Press Enter.
8. Provide invalid value in form field or leave the required field blank, and verify that proper on-screen error identification message is provided and it is programmatically associated for the assistive technology users.

**Actual Result:**
When invalid value is provided in form field or required field is kept blank, then user is not getting an on-screen error message explaining the error that occurred on the page.
Example: "Port" input field.

**Expected Result:**
Whenever an invalid value is provided for the form field, there should be an on-screen message like: Invalid value is provided for the Input Field (Name of the field).
Similarly if required field is kept blank, then there should be an on-screen error message like: This field (Name of the input field) is required, provide appropriate value.

**User Impact:**
Providing information about input errors in text allows users who are blind or colorblind to perceive the fact that an error occurred.
This Success Criterion may help people with cognitive, language, and learning disabilities who have difficulty understanding the meaning represented by icons and other visual cues.

**Note**: Similar issue is observed wherever such scenarios are there throughout the application.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={733120d0-56c5-479c-889f-fee36ea08ee0}

**Attachment:**
[Bug 148_Error identification issue.pptx](https://github.com/Microsoft/IIS.WebManager/files/2330980/Bug.148_Error.identification.issue.pptx)

**Comment by jhkimnew** [comment]
This issue will be fixed when we use WAC SDK control.

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
